Drop artificially created `ping_time` dimension

leewujung opened the pull request:
This PR addresses #1057 by removing `ping_time` dimension artificially added to some variables in `set_groups_*`.

The PR also adds comments on sections of test functions where explicit `ping_time` dim drop is required.

emiliom commented on echopype/tests/calibrate/test_cal_params_integration.py:
```diff
@@ -79,6 +79,8 @@ def test_cal_params_intake_EK60(ek60_path):
     cal_obj = ep.calibrate.calibrate_ek.CalibrateEK60(echodata=ed, env_params=None, cal_params=None, ecs_file=None)
 
     # Check cal params ingested from both ways
+    # Need to drop ping_time for cal_obj.cal_params since get_vend_cal_params_power are related to
```
Not so much being picky as confirming that something isn't missing in this comment.
```suggestion
    # Need to drop ping_time for cal_obj.cal_params since get_vend_cal_params_power is related to
```

emiliom commented on echopype/convert/set_groups_ek60.py:
```diff
@@ -29,23 +29,14 @@ class SetGroupsEK60(SetGroupsBase):
     # the SONAR-netCDF4 v1 convention. For the time being, we are retaining the
     # infrastructure that adds this dimension, but updating the variables lists.
 
+    # 2023-07-21 note: We are also removing the ping_time dimension for parameters
```
This comment seems unnecessary. Otherwise an analogous comment should be added to the EK80 and AZFP modules, and for the removal of the beam dimension.

leewujung replied:
I added this because you added something like that just above this line. I was thinking this should actually be in set_groups_base. I'll move both comments to the leftover mechanism (which is entirely bypassed at this point).

emiliom replied:
Did I? Sigh. It may have been a temporary note to self that I forgot to remove. I agree that it makes more sense to have something like that in `set_groups_base` or elsewhere. Thanks.

emiliom reviewed (commented):
Looks good! My comments are only about some of the comments you added.

leewujung commented:
Sound good. I think I've addressed your comments and will merge after the tests run through (though I did not make any code changes in the new commits).
